packages/pn-personafisica-webapp/src/component/Contacts/DigitalContactsCodeVerification.context.tsx

```diff
@@ -302,14 +300,20 @@ const DigitalContactsCodeVerificationProvider: FC<ReactNode> = ({ children }) =>
                 {t(`${modalProps.labelRoot}.${modalProps.labelType}-new-code`, { ns: 'recapiti' })}
                 &nbsp;
               </Typography>
-              <ButtonNaked
+              <Box
```

Comment:
For the accessibility this can be a problem. Use ButtonNaked but set the property "component" as "Box"